Sources/Plasma/FeatureLib/pfPython/pyVault.cpp

```diff
@@ -412,16 +400,12 @@ PyObject* pyVault::GetPsnlAgeSDL() const
 
     if (hsRef<RelVaultNode> rvnFldr = VaultGetAgesIOwnFolder()) {
 
-        templateNode->ClearFieldFlags();
```

Comment:
I see -- the fact that `templateNode` is unused prior to this line is cut off right before the diff :(